src/language-html/parser-html.js

```diff
@@ -69,9 +69,18 @@ function normalizeAngularControlFlowBlock(node) {
 }
 
 function normalizeAngularLetDeclaration(node) {
-  if (node.type === "letDeclaration") {
-    node.type = "angularLetDeclaration";
+  if (node.type !== "letDeclaration") {
+    return;
   }
+
+  // Similar to `VariableDeclarator` in estree
+  node.type = "angularLetDeclaration";
+  node.id = node.name;
+  node.init = {
+    type: "angularLetDeclarationExpression",
+    sourceSpan: new ParseSourceSpan(node.valueSpan.start, node.valueSpan.end),
+    value: node.value,
+  };
```

Comment:
@sosukesuzuki By change the AST shape to this, so we can print the `.value` in `embeded`.

Reply:
Looks good, but `angularLetDeclarationExpression` is little confusing to me. What about `angularLetDeclarationInitializer` or `angularLetInitializerExpression` ?

Reply:
Also, I am concerned that the fact that we are left with `node.name` and `node.value` instead of just `node.id` and `node.init` may reduce future debuggability.